functions/invoke-d365dbsync.ps1

```diff
@@ -138,12 +138,6 @@ function Invoke-D365DBSync {
     $param += " -metadatabinaries=`"$MetadataDir`""
     $param += " -connect=`"server=$DatabaseServer;Database=$DatabaseName; User Id=$SqlUser;Password=$SqlPwd;`""    
 
-    Write-PSFMessage -Level Debug -Message "Testing if the path exists or not." -Target $LogPath
-    if ((Test-Path -Path $LogPath.Trim() -PathType Leaf) -eq $false) {
-        Write-PSFMessage -Level Debug -Message "Creating the path." -Target $LogPath
-        $null = New-Item -Path $LogPath -ItemType directory -Force -ErrorAction Stop
-    }
-
```

Comment:
I didn't intend for removing this segment of code. Used the editor in the browser, which was a clumsy experience. Disregard the pull on this file, please.